.gitignore

```diff
@@ -159,4 +159,6 @@ override.tf.json
 .terraformrc
 terraform.rc
 
-.terraform.lock.hcl
\ No newline at end of file
+.terraform.lock.hcl
+#ignore local secrets
+todo_app/secret.yaml
```

Comment:
It would be nice to include a copy of this file without any sensitive values in (like the `.env.template`) to make this easy for future developers to set up